frontend/src/components/ProjectMemberPanel.vue

```diff
@@ -1,9 +1,85 @@
 <template>
-  <div>
-    <p class="text-lg font-medium leading-7 text-main">
+  <div class="content-center">
+    <span class="text-lg font-medium leading-7 text-main">
       {{ $t("project.settings.manage-member") }}
-    </p>
-    <div v-if="allowAddMember" class="mt-4 w-full flex justify-start">
+    </span>
+
+    <div v-if="allowSyncVCS" class="inline-block float-right">
+      <span class="normal-link text-sm mr-5" @click="openWindowForVCSMember">
+        {{ $t("project.settings.view-vcs-member") }}
+      </span>
+
+      <span class="text-sm text-control">
+        {{ $t("project.settings.sync-from-vcs") }}
+      </span>
+      <heroicons-outline:refresh
+        v-if="project.roleProvider !== 'BYTEBASE'"
+        class="ml-1 inline text-sm normal-link"
+        @click.prevent="
+          () => {
+            syncMemberFromVCS();
+          }
+        "
+      />
+      <div class="w-auto ml-3 inline-block align-middle">
+        <BBSwitch
+          :value="project.roleProvider !== 'BYTEBASE'"
+          @toggle="
+            (on) => {
+              if (on) {
+                // switching role provider to VCS may result in the deletion of all the existing members.
+                // so we prompt a modal to let user double confirm this.
+                state.showModal = true;
+              } else {
+                // switching role provider back to BYTEBASE will not bring any side effect, so we will prompt nothing.
+                patchProjectRoleProvider('BYTEBASE').then(() => {
+                  store
+                    .dispatch('notification/pushNotification', {
+                      module: 'bytebase',
+                      style: 'SUCCESS',
+                      title: t(
+                        'project.settings.switch-role-provider-to-bytebase-success-prompt'
+                      ),
+                    })
+                    .catch(() => {}); // mute error at browser
+                });
+              }
+            }
+          "
+        />
+      </div>
+    </div>
+
+    <n-modal
```

Comment:
Using kebab-case in templates like <n-modal> seems to be not that consistent with our overall code style.

BTW, seems that we should update BBModal to provide more feature like positive/negative button and events. 😂 Or refactor BBModal to use NModal internally.

Reply:
Maybe we need to put this into our backlog?

Reply:
PTAL